examples/platform/esp32/common/InitServerHelpers.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/server/AppDelegate.h>
+#include <stdint.h>
+
+class InitServerHelper
+{
+
+public:
+    static void Init(void (*callback)() = nullptr, AppDelegate * context = nullptr);
```

Comment:
If the callback happens always at the end of the `Init` function, maybe we can remove it and have the users execute the code themselves?

Reply:
Removed callback.